web-app/src/routes/[event]/auth/_layout.svelte

```diff
@@ -34,7 +36,7 @@
 </style>
 
 <main class="MAIN">
-  <img class="IMAGE" alt="Cheers" src="social-beer--green.png">
+  <img class="IMAGE" alt="Cheers" src={Icon}>
```

Comment:
Should have double quotes for consistency
```suggestion
  <img class="IMAGE" alt="Cheers" src="{Icon}">
```